### ISSUE TITLE
write guide on localized routes

### ISSUE THREAD

**Issue by RWOverdijk** [balderdashy/sails-docs#623]
I've extracted this snippet from the docs [over here](https://github.com/balderdashy/sails-docs/blob/master/concepts/Internationalization/Locales.md) as it was causing visual issues.

### Snippet

  Alternatively, here's another extended example:
  (todo: at the very least pull this into a separate guide)

```js
// config/routes.js
module.export.routes = {
  '/:lang/': 'MyController.index',
  '/:lang/help': 'MyController.help',
  '/:lang/contact': 'MyController.contact',
  // ...etc...
}

// config/policies.js
module.exports.policies = {
  '*' : 'localize'
}

// api/policies/localize.js
module.exports = function(req, res, next) {
   req.setLocale(req.param('lang'));
   next();
};
```

**Comment by mikermcneil** [comment]
@RWOverdijk makes sense--let me dig up the link I'm thinking of that we might want to include real quick and get back to you

**Comment by RWOverdijk** [comment]
ack

**Comment by mikermcneil** [comment]
@RWOverdijk argh nevermind, sorry I can't find it.  Someone started a PR doing something related w/ localized routes (it was technically a different thing-- like the section on "internationalization in url patterns" [here](https://docs.djangoproject.com/es/1.9/topics/i18n/translation/) --  but both could fit in the same docs section).

Anyways, you're right, let's set up a doc page for it in the same section ([like this one](https://github.com/balderdashy/sails-docs/blob/master/concepts/Internationalization/TranslatingDynamicContent.md)).  How about "Language Route Prefixes"?  (or "route prefixes" if that wouldn't fit-- a la what django does: https://docs.djangoproject.com/es/1.9/topics/i18n/translation/)

And thanks for pulling that out, btw.

**Comment by sailsbot** [comment]
Thanks for posting, @RWOverdijk.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
